modules/apps/commerce/commerce-product-content-search-web/src/main/java/com/liferay/commerce/product/content/search/web/internal/display/context/builder/CPSpecificationOptionsFacetDisplayContextBuilder.java

```diff
@@ -376,17 +370,10 @@ private boolean _isCPDefinitionSpecificationOptionValueSelected(
 		CPSpecificationOption cpSpecificationOption = _getCPSpecificationOption(
 			fieldName);
 
-		Optional<String[]> parameterValuesOptional =
+		return ArrayUtil.contains(
```

Comment:
@julschong Unlike https://github.com/liferay-core-infra/liferay-portal/pull/4529/files#r1209733811, it's ok to do this here, because `ArrayUtil.contains` has built-in null check.